lib/utils.js

```diff
@@ -38,10 +38,11 @@ function newGuid () {
     s4() + '-' + s4() + s4() + s4();
 }
 
-function sanitizeName (name) {
+function sanitizeName(name) {
   var sanitizedName = name;
-  
+
   // Remove all invalid characters
+  sanitizedName = sanitizedName.replace(/[^a-zA-Z0-9]/g, '_');
```

Comment:
migrating logic used in the pwabuilder-api level to here:
https://github.com/pwa-builder/pwabuilder-api/blob/8c5711e758f5021f1dc3535659ad861867685dc2/src/services/storage.js#L38